components/Collections/src/MDCCollectionViewController.m

```diff
@@ -434,12 +434,12 @@ - (BOOL)collectionView:(UICollectionView *)collectionView
 
 - (void)collectionView:(UICollectionView *)collectionView
     didSelectItemAtIndexPath:(NSIndexPath *)indexPath {
-  [self updateFooterInfoBarIfNecessary];
+  [_footerInfoBar showAnimated:YES];
 }
 
 - (void)collectionView:(UICollectionView *)collectionView
     didDeselectItemAtIndexPath:(NSIndexPath *)indexPath {
-  [self updateFooterInfoBarIfNecessary];
+  [_footerInfoBar dismissAnimated:YES];
```

Comment:
Is this correct? It seems like selecting multiple items is supported from the implementation of updateFooterInfoBarIfNecessary. If the user selects two items and then deselects one of them, this will dismiss the footerInfoBar even though at least one other item is selected.